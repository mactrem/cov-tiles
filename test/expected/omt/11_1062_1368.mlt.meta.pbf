F
	landcover
id" 
geometry* 
class" 	
subclass" 	É
aerodrome_label
id" 
geometry* 
class" 	
ele" 
ele:ft" 
iata" 	
icao" 	
name" 	
name:de" 	
name:en" 	
name:int" 	

name:latin" 	t
waterway
id" 
geometry* 
class" 	
name" 	
name:de" 	
name:en" 	
name:nl" 	2
landuse
id" 
geometry* 
class" 	¹
mountain_peak
id" 
geometry* 
class" 	
ele" 
ele:ft" 
name" 	
name:de" 	
name:en" 	
name:int" 	

name:latin" 	
rank" Ø
place
id" 
geometry* 
class" 	
name" 	
name:de" 	
name:en" 	
name:int" 	

name:latin" 	
rank" 
name:ru" 	
name:ar" 	
name:la" 	
name:sr" 	F
water
id" 
geometry* 
class" 	
intermittent" p
transportation_name
id" 
geometry* 
class" 	
network" 	
ref" 	

ref:length" “
park
id" 
geometry* 
class" 	
name" 	
name:de" 	
name:en" 	
name:int" 	

name:latin" 	
rank" ?
aeroway
id" 
geometry* 
class" 	
ref" 	–
transportation
id" 
geometry* 
class" 	
oneway" 
ramp" 
bicycle" 	
foot" 	
brunnel" 	
layer" 