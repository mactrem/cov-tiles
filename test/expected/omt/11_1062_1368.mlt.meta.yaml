version: 1
featureTables {
  name: "landcover"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "subclass"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "aerodrome_label"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "ele"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "ele:ft"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "iata"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "icao"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:de"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:en"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:int"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:latin"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "waterway"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:de"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:en"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:nl"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "landuse"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "mountain_peak"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_64
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "ele"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "ele:ft"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "name"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:de"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:en"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:int"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:latin"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "rank"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
}
featureTables {
  name: "place"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_64
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:de"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:en"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:int"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:latin"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "rank"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "name:ru"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:ar"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:la"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:sr"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "water"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "intermittent"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
}
featureTables {
  name: "transportation_name"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "network"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "ref"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "ref:length"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
}
featureTables {
  name: "park"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:de"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:en"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:int"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:latin"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "rank"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
}
featureTables {
  name: "aeroway"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "ref"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "transportation"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "oneway"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "ramp"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "bicycle"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "foot"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "brunnel"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "layer"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
}
